tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -658,9 +658,25 @@ async function cleanCompassResourcesSKR(client, appName, scenarioName, runtimeID
     console.log(err);
   }
 }
+const defaultRetryDelayMs = 1000;
+
+
+const retryWithDelay = (operation, delay, retries) => new Promise((resolve, reject) => {
```

Comment:
it seems the function is duplicated. Can you not re-use [this](https://github.com/hisarbalik/kyma/blob/ee43f2fe3d6c952e881db648657fa7b9923dacfc/tests/fast-integration/utils/index.js#L78)? 
If it is different, is it possible to put it in a central place as the linked function? You have defined the function `retryWithDelay` twice in different files.

Reply:
duplicate removed